<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btSettings">
     <property name="text">
      <string>&amp;Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btCon">
     <property name="text">
      <string>&amp;Conectar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btDesc">
     <property name="text">
      <string>&amp;Desconectar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btProc">
     <property name="text">
      <string>&amp;Procurar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btSniff">
     <property name="text">
      <string>S&amp;niffer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Desconectado</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>hex</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
